lib/src/view/game/game_result_dialog.dart

```diff
@@ -97,7 +101,34 @@ class _GameResultDialogState extends ConsumerState<GameResultDialog> {
                             gameState.game.opponent?.onGame == true &&
                             gameState.game.opponent?.offeringRematch != true
                         ? () {
-                            ref.read(ctrlProvider.notifier).proposeOrAcceptRematch();
+                            final game = gameState.game;
+
+                            // Check if it was an Odds Bot game
```

Comment:
This comment is useless and misleading because `fromPosition` doesn't necessarily mean an odds bot game.